ads/inabox/position-observer.js

```diff
@@ -179,3 +182,14 @@ function getScrollingElement(win) {
 function isWebKit(ua) {
   return /WebKit/i.test(ua) && !/Edge/i.test(ua);
 }
+
+/**
+ * Use an existing position observer within the window, if any.
+ * @param {!Window} win
+ * @return {!PositionObserver}
+ */
+export function getPositionObserver(win) {
```

Comment:
this seems only needed in host.js? we can move it over there.

Reply:
It's used in `src/inabox/inabox-viewport.js` as well. It could be kept here.